src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -868,6 +884,14 @@ static DistributeObjectOps Type_Rename = {
 	.address = RenameTypeStmtObjectAddress,
 	.markDistributed = false,
 };
+static DistributeObjectOps View_Rename = {
```

Comment:
There seems to be some issues with `IF EXISTS` in general.

Citus:
```SQL
 ALTER VIEW IF EXISTS v1 rename to t1;
ERROR:  relation "v1" does not exist
```

PG:
```
 ALTER VIEW IF EXISTS v1 rename to t1;
NOTICE:  relation "v1" does not exist, skipping
ALTER VIEW
Time: 1.604 ms
```

Reply:
Similarly for set schema, this should not send the command to the workers:
```SQL
alter view if exists sc10."r1 owned view renamed" set schema "r1 owned schama" ;
NOTICE:  relation "r1 owned view renamed" does not exist, skipping
ERROR:  relation "sc10.r1 owned view renamed" does not exist
CONTEXT:  while executing command on localhost:9705
Time: 35.567 ms
```

Can you maybe add set of `IF EXISTS` tests to non-existing tables for all the sub-commands?